### PR TITLE
fixed error range for duplicate unique values

### LINKED ISSUE #1210: Incorrect error range for `cvc-identity-constraint.4.1`
Given this XML file:

```xml
<?xml version="1.0" encoding="UTF-8" ?>
<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xmlns="http://java.sun.com/xml/ns/javaee"
    xmlns:web="http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
    xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
    id="WebApp_ID" version="2.5"> 
    
		<servlet>
        <servlet-name>dispatcher</servlet-name>
        <servlet-class>
            org.springframework.web.servlet.DispatcherServlet
        </servlet-class>
        <load-on-startup>1</load-on-startup>
    </servlet>

		<servlet>
			<servlet-name>dispatcher</servlet-name>
			<servlet-class>
					org.springframework.web.servlet.DispatcherServlet
			</servlet-class>
			<load-on-startup>1</load-on-startup>
	</servlet>
	
</web-app>
```

The error range is not correct (at end of the servlet-name) :

image

The error range should higlight the `dispatcher` text node.

### PR DESCRIPTION
**Pull request by JessicaJHee**
image
Fixes #1210 

Signed-off-by: Jessica He <jhe@redhat.com>

### CODE REVIEW

File: org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/XMLSchemaErrorCode.java [hunk 1]
```diff
@@ -83,6 +83,7 @@ public enum XMLSchemaErrorCode implements IXMLErrorCode {
 	cvc_maxInclusive_valid("cvc-maxInclusive-valid"), // https://wiki.xmldation.com/Support/validator/cvc-maxinclusive-valid
 	cvc_minExclusive_valid("cvc-minExclusive-valid"), // https://wiki.xmldation.com/Support/validator/cvc-minexclusive-valid
 	cvc_minInclusive_valid("cvc-minInclusive-valid"), // https://wiki.xmldation.com/Support/validator/cvc-mininclusive-valid
+	cvc_identity_constraint_4_1("DuplicateUnique"), //
```
[the remark this answers is not in the document]
Comment (AlexXuChen) [reply org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/XMLSchemaErrorCode.java:86]: It seems the original issue also shows the error code as `DuplicateUnique`: https://github.com/eclipse/lemminx/issues/1210#issue-1253908887

[the remark this answers is not in the document]
Comment (JessicaJHee) [reply org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/XMLSchemaErrorCode.java:86]: image

I can either leave it, change it to `DuplicateUnique("DuplicateUnique")` or try to change the original error code being sent to `cvc_identity_constraint_4_1`. Not sure what's best here?

[the remark this answers is not in the document]
Comment (angelozerr) [reply org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/XMLSchemaErrorCode.java:86]: Ok I understand more the problem. The real XSD error code should be `cvc-identity-constraint-4-1`
https://www.w3.org/TR/xmlschema11-1/#cvc-identity-constraint 
but Xerces uses the custom DuplicateUnique code https://github.com/apache/xerces2-j/blob/cf0c517a41b31b0242b96ab1af9627a3ab07fcd2/src/org/apache/xerces/impl/msg/XMLSchemaMessages.properties#L37

I suggest that you change with `DuplicateUnique("DuplicateUnique")` to be aligned with the Xerces error code and add a comment that this error code is for cvc-identity-constraint.4.1.

**Comment by angelozerr** [comment]
@JessicaJHee it works like a charm, thanks!

**Pull request merged**